internal/http/services/owncloud/ocdav/put.go

```diff
@@ -120,6 +121,15 @@ func (s *svc) handlePut(w http.ResponseWriter, r *http.Request, ns string) {
 	}
 
 	if ok {
+		// TODO: disable if chunking capability is turned off in config
```

Comment:
just remove the code

Reply:
assuming you meant remove all the commented out code and don't add a check ? (not remove the chunking code)